extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -820,23 +827,22 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 				async () => {
 					if (this.rowsFilegroupsTable.selectedRows.length === 1) {
 						const fileGroup = this.rowDataFileGroupsTableRows[this.rowsFilegroupsTable.selectedRows[0]];
-						await this.rowsFilegroupNameInput.updateCssStyles({ 'visibility': fileGroup.id < 0 ? 'visible' : 'hidden' });
+						await this.rowsFilegroupNameContainer.updateCssStyles({ 'visibility': fileGroup.id < 0 ? 'visible' : 'hidden' });
```

Comment:
You can remove this container from the DOM by using "display: none" instead of "visibility: hidden". That way you won't have to do constant margin adjustments on the buttons below it

Reply:
I do a similar thing with hiding controls in the attach databases dialog. https://github.com/microsoft/azuredatastudio/blob/main/extensions/mssql/src/objectManagement/ui/attachDatabaseDialog.ts#L83

Reply:
Let me try it :)

the last time when i used display, the dialog display property is overriding it and had to use visibility.

Reply:
it worked! thanks